__tests-pacts__/serlo.org/user.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * This file is part of Serlo.org API
+ *
+ * Copyright (c) 2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/api.serlo.org for the canonical source repository
+ */
+import { Matchers } from '@pact-foundation/pact'
+import fetch from 'node-fetch'
+
+import { addJsonInteraction } from '../__utils__'
+
+test('list of active author ids', async () => {
+  await addJsonInteraction({
+    name: 'fetch list of active author ids',
+    given: 'users with ids 1 and 10 are active authors',
+    path: '/api/user/active-authors',
+    body: Matchers.like([1, 10]),
```

Comment:
That won't work, unfortunately. Pact is really bad with handling arrays (so here you are actually writing that you expect the endpoint to respond with exactly two users, which won't be true most of the time). Instead:
```suggestion
    body: Matchers.eachLike(1),
```
tells Pact that you expect a (non-empty) number array.